specification/data-resource-semantic-conventions.md

```diff
@@ -70,6 +70,28 @@ Labels defining a compute unit (e.g. Container, Process, Lambda Function).
 | container.image.name | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` |
 | container.image.tag | Container image tag. | `0.1` |
 
+
+### Function as a Service
+
+**type:** `faas`
+
+**Description:** A serverless instance.
```

Comment:
Because serverless is more of a buzzwordy term and not very well defined, I would try to think of some alternative phrasing.
```suggestion
**Description:** An instance of a function that runs on a platform where underlying hardware concerns such as provisioning and scaling have been abstracted away by the platform.
```

Reply:
You could still add (aka "serverless").

Reply:
I agree, otherwise totally different products, such as knative, would be put into this category.